### PR TITLE
feat: delete outdated tensorboard experiments

### PR DESCRIPTION
**Pull request by inardini**
<br>
This notebooks shows how to delete outdated TensorBoard Experiments to avoid unnecessary storage costs.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb [hunk 1]
```diff
@@ -0,0 +1,667 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>Heading must be in sentence case</p><p> </p><p>Delete outdated Vertex AI TensorBoard Experiment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>TensorBoard -&gt; Vertex AI TensorBoard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>TensorBoard Experiments -&gt; Vertex AI TensorBoard Experiments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>has been -&gt; is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>replace "ie" with ":"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <h2>Tensorboard Experiments -&gt; Vertex AI TensorBoard Experiments</h2><h2><br></h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <h3>TensorBoard experiments  -&gt; TensorBoard Experiments</h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>Tensorboard -&gt; TensorBoard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>Tensorboard -&gt; TensorBoard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <h3><strong>Tensorboard -&gt; TensorB</strong>oard</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <h3><strong>Tensorboard -&gt; TensorB</strong>oard</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb:1]: <p>done</p>

**Pull request merged**